flink/v1.14/flink/src/main/java/com/netease/arctic/flink/lookup/RocksDBCacheState.java

```diff
@@ -71,6 +71,7 @@ public abstract class RocksDBCacheState<V> {
   protected BinaryRowDataSerializerWrapper valueSerializer;
   private ExecutorService writeRocksDBService;
   private final AtomicBoolean initialized = new AtomicBoolean(false);
+  private final AtomicBoolean running = new AtomicBoolean(true);
```

Comment:
Is the variable name `closed` better here?
I do not think the class is runnable.

Reply:
Make sense to me.